packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -1749,7 +1761,7 @@ exports.questions = {
 			},
 			{
 				text: 'Not started yet',
-				value: 'not started',
+				value: 'not-started',
```

Comment:
If this is given as the answer, does it appear on the task list page as 'Not-started' with the hyphen, is this the intended appearance?

Only asking as the radio question i did with 'written representation' i didnt put a hypen in as it looked strange on the task list page.

Reply:
Good spot, fixed it now